cmd/create/cluster/cmd.go

```diff
@@ -1150,20 +1149,20 @@ func run(cmd *cobra.Command, _ []string) {
 					billingAccount = aws.ParseOption(billingAccount)
 				}
 
-				err := validateBillingAccount(billingAccount)
+				err := ocm.ValidateBillingAccount(billingAccount)
 				if err != nil {
 					r.Reporter.Errorf("%v", err)
 					os.Exit(1)
 				}
 
 				// Get contract info
-				contracts, isContractEnabled := GetBillingAccountContracts(cloudAccounts, billingAccount)
+				contracts, isContractEnabled := ocm.GetBillingAccountContracts(cloudAccounts, billingAccount)
 
 				if billingAccount != awsCreator.AccountID {
 					r.Reporter.Infof(
-						"The selected AWS billing account is a different account than your AWS infrastructure account." +
+						"The selected AWS billing account is a different account than your AWS infrastructure account. " +
 							"The AWS billing account will be charged for subscription usage. " +
-							"The AWS infrastructure account will be used for managing the cluster.",
+							"The AWS infrastructure account contains the ROSA infrastructure.",
```

Comment:
the _account_ contains the infrastructure? or does the thing that the account is _for_ contain the infrastructure?

Reply:
@jseseCCS it's the latter.